Python/flowgraph.c

```diff
@@ -531,7 +531,7 @@ normalize_jumps_in_block(cfg_builder *g, basicblock *b) {
     if (backwards_jump == NULL) {
         return ERROR;
     }
-    basicblock_addop(backwards_jump, JUMP, target->b_label.id, NO_LOCATION);
+    basicblock_addop(backwards_jump, JUMP, target->b_label.id, last->i_loc);
```

Comment:
This is the last physical location, not the last location executed. Won't it give the wrong line for the following?
```Py
ALWAYS_TRUE= True

def foo():
    while 1:
        if ALWAYS_TRUE:
            pass # last line executed
        else:
            pass # last physical line
```

Reply:
no, last is the forward jump that this jump is replacing.